web3/utils/module_testing/parity_module.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+
+from eth_utils import (
+    add_0x_prefix,
+)
+
+from web3.utils.formatters import (
+    hex_to_integer,
+)
+
+
+class ParityModuleTest:
+
+    def test_trace_replay_transaction(self, web3, parity_fixture_data):
+        trace = web3.parity.traceReplayTransaction(parity_fixture_data['mined_txn_hash'])
+
+        assert trace['stateDiff'] is None
+        assert trace['vmTrace'] is None
+        assert trace['trace'][0]['action']['from'] == add_0x_prefix(parity_fixture_data['coinbase'])
+
+    def test_trace_replay_block_transactions(self, web3, block_with_txn):
+        web3.parity.traceReplayBlockTransactions(block_with_txn['number'])
```

Comment:
Is there anything we can assert about how this should work in newer Parity versions?

Reply:
And Done!